mitmproxy/tools/web/static_viewer.py

```diff
@@ -54,7 +54,7 @@ def save_flows_content(self) -> None:
                 path = os.path.join(self.path, 'flows', f.id, m)
                 if not os.path.exists(path):
                     os.makedirs(path)
-                with open(os.path.join(path, 'content.json'), 'wb') as content_file:
+                with open(os.path.join(path, '_content'), 'wb') as content_file:
```

Comment:
Change `content.json` to `_content` here, or it will be weired to access `<flow-id>/response/content.json` to get a picture file. We could not specify the download file name for the user in static mode now, so use this temporary solution.